dags/socorro_import.py

```diff
@@ -18,39 +39,155 @@
+# TODO - change this to just s3://telemetry-parquet when ready
+dest_s3_key="s3://telemetry-parquet/socorro_crash_hwoo"
+
+# We don't use -m for parallel rsync because I ran into 503 while issues using it.
+gsutil_args = [
+    'gsutil', 'rsync', '-r',
+    'gs://{}/{}'.format(gcs_data_bucket, objects_prefix),
+    '{}/{}'.format(dest_s3_key, objects_prefix)
+]
+
+# Copy parquet data from gcs to s3
+gcs_to_s3 = GKEPodOperator(
+    task_id='gcs_to_s3',
+    gcp_conn_id=bq_gcp_conn_id,
+    project_id=bq_connection.project_id,
+    location=gke_location,
+    cluster_name=gke_cluster_name,
+    name='copy-parquet-data-back-to-s3',
+    namespace='default',
+    image="google/cloud-sdk:242.0.0-alpine",
+    arguments=gsutil_args,
+    env_vars={
+        "AWS_ACCESS_KEY_ID": "{{ var.value.socorro_access_key_id }}",
+        "AWS_SECRET_ACCESS_KEY": "{{ var.value.socorro_secret_access_key }}"
+    },
+    dag=dag,
+)
+
+# Not using load_to_bigquery since our source data is s3 json, instead of parquet
+# We do still use the parquet2bigquery container to load parquet into bq.
+# TODO _ change the dataset also to telemetry?
+bq_dataset='tmp'
+bq_table_name = '{}_{}'.format(dataset, dataset_version)
+
+docker_image='docker.io/mozilla/parquet2bigquery:20190722' # noqa
+
+gke_args = [
+    '-d', bq_dataset,
+    '-c', '10',
+    '-b', gcs_data_bucket,
+    '-R',
+    '-p', objects_prefix,
+    '--cluster-by', 'crash_date'
+]
+
+remove_bq_table_partition = BigQueryTableDeleteOperator(
+    task_id='remove_bq_table_partition',
+    bigquery_conn_id=bq_gcp_conn_id,
+    deletion_dataset_table='{}.{}${{{{ds_nodash}}}}'.format(bq_dataset, bq_table_name), # noqa
+    ignore_if_missing=True,
+    dag=dag
+)
+
+bq_load = GKEPodOperator(
+    task_id='bigquery_load',
+    gcp_conn_id=bq_gcp_conn_id,
+    project_id=bq_connection.project_id,
+    location=gke_location,
+    cluster_name=gke_cluster_name,
+    name='load-socorro-crash-parquet-to-bq',
+    namespace='default',
+    image=docker_image,
+    arguments=gke_args,
     dag=dag,
 )
 
 register_status(
-    crash_report_parquet,
-    crash_report_parquet.job_name,
+    bq_load,
+    "Socorro Crash Reports Parquet",
     "Convert processed crash reports into parquet for analysis",
 )
 
-crash_report_parquet_bigquery_load = SubDagOperator(
-    subdag=load_to_bigquery(
-        parent_dag_name=dag.dag_id,
-        dag_name="crash_report_parquet_bigquery_load",
-        default_args=default_args,
-        dataset_s3_bucket="telemetry-parquet",
-        aws_conn_id="aws_dev_iam_s3",
-        dataset="socorro_crash",
-        dataset_version="v2",
-        date_submission_col="crash_date",
-        gke_cluster_name="bq-load-gke-1",
-        bigquery_dataset="telemetry_derived",
-        ),
-    task_id="crash_report_parquet_bigquery_load",
-    dag=dag)
-
-crash_report_parquet >> crash_report_parquet_bigquery_load
+s3_to_gcs >> crash_report_parquet >> gcs_to_s3 >> remove_bq_table_partition >> bq_load
```

Comment:
```suggestion
s3_to_gcs >> crash_report_parquet 
crash_report_parquet >> gcs_to_s3 
crash_report_parquet >> remove_bq_table_partition >> bq_load
```

It doesn't look like there's a dependency between the bq load and the gcs to s3 transfer.

Reply:
thanks good catch